src/ci/scripts/install-msys2-packages.sh

```diff
@@ -13,6 +13,7 @@ if isWindows; then
     # one way or another. The msys interpreters seem to have weird path conversions
     # baked in which break LLVM's build system one way or another, so let's use the
     # native version which keeps everything as native as possible.
-    cp C:/Python27amd64/python.exe C:/Python27amd64/python2.7.exe
-    ciCommandAddPath "C:\\Python27amd64"
+    ls C:/hostedtoolcache/windows/Python/2.7.17/python.exe
+    cp C:/hostedtoolcache/windows/Python/2.7.17/python.exe C:/hostedtoolcache/windows/Python/2.7.17/python2.7.exe
```

Comment:
I can't say that this directory name exactly inspires confidence :)

Reply:
Heh, at least [it's somewhat documented](https://github.com/actions/virtual-environments/blob/master/images/win/Windows2016-Readme.md#python).